pom.xml

```diff
@@ -94,7 +94,7 @@ under the License.
     <maven.compiler.source>${maven.compiler.target}</maven.compiler.source>
     <maven.compiler.target>1.7</maven.compiler.target>
     <surefire.version>2.22.2</surefire.version><!-- for surefire, failsafe and surefire-report -->
-    <maven.plugin.tools.version>3.6.4</maven.plugin.tools.version><!-- for m-plugin-p and maven-plugin-annotations -->
+    <maven.plugin.tools.version>3.7.0</maven.plugin.tools.version><!-- for m-plugin-p and maven-plugin-annotations -->
```

Comment:
Please in addition also manage the version of the new maven-plugin-report-plugin